keanu-project/src/test/java/io/improbable/keanu/e2e/foodpoisoning/FoodPoisoningTest.java

```diff
@@ -54,10 +59,11 @@ public void oystersAreInfected() {
     @Category(Slow.class)
     @Test
     public void lambAndOystersAreInfected() {
-        generateSurveyData(50, true, true, false);
-        NetworkSamples samples = sample(15000);
+        generateSurveyData(30, true, true, false);
+
+        int dropCount = 2000;
+        NetworkSamples samples = sample(5000).drop(dropCount);
 
-        int dropCount = 10000;
```

Comment:
Good spot that the `dropCount` isn't being used in a few tests here.

Reply:
👍